src/main/java/Mapper/GameMapper.java

```diff
@@ -0,0 +1,26 @@
+package Mapper;
+
+import com.example.gamelist.Game;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.awt.event.PaintEvent.UPDATE;
+import static javax.swing.text.html.parser.DTDConstants.ID;
+
+@Mapper
+public interface GameMapper {
+
+    //データを一括で取得する。　GET
+    @Select("SELECT * FROM gamelists")
+    List<Game> findAll();
+
+    @Select("SELECT * FROM gamelists WHERE id = #{id}")
+    Optional<Game> findById(int id);
+
+    @Update(UPDATE gamelist SET name = {#name} WHERE id = #{id})
```

Comment:
`@Select`の書き方をみてもらうとわかるのですが、ダブルクオートで囲まないといけません。
```suggestion
    @Update("UPDATE gamelist SET name = {#name} WHERE id = #{id}")
```

Reply:
<img width="612" alt="スクリーンショット 2023-12-16 23 25 38" src="https://github.com/otyacazyu/gamelist/assets/143002834/c5bd7044-958f-4acc-bfa8-1ce20140a507">

編集を行いました。
教えてくださり、ありがとうございます。

Reply:
コミットハッシュも貼りましょう！

Reply:
コミットハッシュを貼らして頂きます。

[045311d](https://github.com/otyacazyu/gamelist/pull/7/commits/045311dbdcf6e64681490c71bdb50ad16f570b5e)

Reply:
OKです。